wave_utils/include/wave/utils/math.hpp

```diff
@@ -37,6 +37,9 @@ void mat2vec(MatX A, std::vector<double> &x);
 double wrapTo180(double euler_angle);
 double wrapTo360(double euler_angle);
 int euler2rot(Vec3 euler, int euler_seq, Mat3 &R);
+int euler2quat(Vec3 euler, int euler_seq, Quaternion &q);
```

Comment:
Required by the `Pose` class, since it stores a Quaternion. I don't know whether it is a good idea to use `Rotation` in geometry or keep it as a `Eigen::Quaternion`, if so the kinematics will have a strong dependence on `geometry` module.